File: pbo.java

import java.util.Scanner;
/* PROGRAM PENJUMLAHAN MATRIKS*/
class PenjumlahanMatriks {
    public static void main(String[] args){
            Scanner input = new Scanner(System.in);
            int A[][] = new int[2][2];
            int B[][] = new int[2][2];
            int C[][] = new int[2][2];
            String myText = "MANTAB";

            System.out.println("Masukkan Matriks A");
            for (int i = 0; i < 2; i++) {
                for (int j = 0; j < 2; j++) {
                    System.out.print("[" + (i + 1) + "][" + (j + 1) + "]:");
                    A[i][j] = input.nextInt();
                }
            }
            System.out.println("Masukkan Matriks B");
            for (int k = 0; k < 2; k++) {
                for (int l = 0; l < 2; l++) {
                    System.out.print("[" + (k + 1) + "][" + (l + 1) + "]:");
                    B[k][l] = input.nextInt();
                }
            }
            System.out.println("Matriks A");
            for (int i = 0; i < 2; i++) {
                for (int j = 0; j < 2; j++) {
                    System.out.print(+(A[i][j]) + " ");
                }
                System.out.println(" ");
            }
            System.out.println("Matriks B");
            for (int k = 0; k < 2; k++) {
                for (int l = 0; l < 2; l++) {
                    System.out.print(+(B[k][l]) + " ");
                }
                System.out.println(" ");
            }
            for (int x = 0; x < 2; x++) {
                for (int y = 0; y < 2; y++) {
                    C[x][y] = A[x][y] + B[x][y];
                }
            }
            System.out.println("Hasil penjumlahan Matriks A dan Matriks B");
            for (int x = 0; x < 2; x++) {
                for (int y = 0; y < 2; y++) {
                    System.out.print(+(C[x][y]) + " ");
                }
                System.out.println(" ");
            }
            int x = 0;
                if (x < 2) {
                    System.out.println("PROGRAM BERHASIL BERJALAN");
                } else {
                    System.out.println("PROGRAM TIDAK DAPAT DIJALANKAN");
                }

                int i = 0;
                do{
                    System.out.println("mantab");
                    i++;
                }while (i<3);
        }

}

File: pbo.cpp
#include <iostream>
#include <string>
using namespace std;
/*SEMANGAT NUGAS*/
int main(){
    int i, j;
    int x=0;
int A[2][2], B[2][2], O[2][2];
cout<<"Penjumlahan Matriks 2×2"<<endl;
cout<<"Input elemen matriks: "<<endl;

    for(i=0;i<2;i++){
        for(j=0;j<2;j++){
        cout<<"A["<<(i+1)<<"]["<<(j+1)<<"] = ";
        cin>>A[i][j];
            }
         }

    for(i=0;i<2;i++){
       for(j=0;j<2;j++){
            cout<<"B["<<(i+1)<<"]["<<(j+1)<<"] = ";
            cin>>B[i][j];
            }
        }

for(i=0;i<2;i++){
    for(j=0;j<2;j++){
       O[i][j]=A[i][j]+B[i][j];
    }
}

cout<<"Matriks A: "<<endl;
for(i=0;i<2;i++)
{
for(j=0;j<2;j++)
{
cout<<A[i][j]<<"\t";
}
cout<<endl;
}
cout<<endl;

cout<<"Matriks B:"<<endl;
for(i=0;i<2;i++){
        for(j=0;j<2;j++){
            cout<<B[i][j]<<"\t";
    }
    cout<<endl;
  }
 cout<<endl;

 cout<<"Hasil Penjumlahan Matriks A dan B:"<<endl;
        for(i=0;i<2;i++){
            for(j=0;j<2;j++){
                cout<<O[i][j]<<"\t";
                //cout<<"["<<O[i]<<","<<O[j]<<"]"<<endl;
            }
            cout<<endl;
        }

    if (x<2){
        cout<<"Program bisa dijalankan"<<endl;
    }else{
            cout<<"Program tidak bisa dijalankan"<<endl;

        }

        do{
            cout<<"mantab";
            i++;
        }while (i<3);

return 0 ;
}
